README.md

```diff
@@ -51,6 +51,29 @@ To skip local test
 
 in your terminal. The project will then be compiled and tested before finally being installed in your local repository.
 
+## Making an internal release
```

Comment:
This instruction seems a bit odd here, as this is public repo and will be visible when you open repo...maybe we could "hide" it somewhere? though I'm not sure where 😆